src/test/scala/io/iohk/ethereum/ledger/BlockImportSpec.scala

```diff
@@ -135,37 +135,18 @@ class BlockImportSpec extends FlatSpec with Matchers with MockFactory {
     blockQueue.isQueued(newBlock3.header.hash) shouldBe false
   }
 
-  it should "import blocks out of order" in new TestSetup with EphemBlockchain {
```

Comment:
Is there any test that replaces this one and returns `UnknownParent` instead?

Reply:
There is now, good catch